Gruntfile.js

```diff
@@ -79,7 +88,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-jsxgettext');
   grunt.loadNpmTasks('grunt-zanata-js');
 
-  grunt.registerTask('test', ['fh:test']);
+  grunt.registerTask('test', ['fh:test', 'eslint']);
```

Comment:
I'm not sure why, but this doesn't seem to be running as part of the job ('test' is, but 'eslint' isn't). Does eslint run if you run `grunt test` locally?